final/problem-def-ana/require-ana.tex

```diff
@@ -162,6 +171,8 @@ \subsubsection{Won't have}
 \subsection{Minimal viable product}
```

Comment:
This has also been taken from the research report, so naturally it doesn't add any new content. Do you have any suggestions as to what you'd like to see added?